elements/timeline-view/timeline-dialog.html

```diff
@@ -90,6 +103,23 @@
                 this.preset = activePreset;
                 this.$.presetView.hidden = false;
             }
+            this.$.presets_show.hidden = this.presets.length == 0;
+            this.$.presets_hide.hidden = this.presets.length > 0;
+
+            this.$.cancel_button.hidden = this.presets.length == 0;
+            this.$.accept_button.hidden = this.presets.length == 0;
+            this.$.close_button.hidden = this.presets.length > 0;
```

Comment:
These inline conditionals are repeated. Please consider extracting them to local `const`s.